wordpress-mobile/WordPress-iOS
# UL&S: Add tableView and continue button to SiteAddressViewController

Ref. #14009 
Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/294

This PR points to the latest Auth feature branch and also creates a second instance of `WordPressAuthenticatorStyle` for use with unified auth changes.

### To test
1. Enable the .unifiedSiteAddress feature flag, `return BuildConfiguration.current == .localDeveloper`
2. `rake dependencies`
3. Build and run
4. Log In > "entering your site address" 
**Expected:** a Continue button appears. The tableview isn't visible at this point because it has no content.

<a href="https://user-images.githubusercontent.com/1062444/83460189-6f88b280-a42b-11ea-88e1-e6cee06985de.png"><img src="https://user-images.githubusercontent.com/1062444/83460189-6f88b280-a42b-11ea-88e1-e6cee06985de.png" width="350" /></a>


PR submission checklist:
